entries/r/rmit.edu.au.json

```
@@ -0,0 +1,18 @@
+{
+  "RMIT University": {
+    "domain": "rmit.edu.au",
+    "tfa": [
+      "custom-software"
```

SMS and manual TOTP code entry are available as well according to the doc.
```suggestion
      "sms",
      "totp",
      "custom-software"
```